README.md

```diff
@@ -13,6 +13,10 @@ Tezos Globacap project provides two entities:
 
 There is a [specification](/docs/specification.md) for the required smart contracts.
 
+## Holdings contract documentation
+
+Holdings contract documentation can be found [here](https://github.com/serokell/tezos-globacap/-/blob/autodoc/master/Holdings-contract.md).
```

Comment:
I would make the difference between autodoc and `specification.md` a bit clearer, as we did it in another repo (a private one).

Reply:
updated, btw